src/generated/resources/openapi.json

```diff
@@ -767,14 +767,19 @@
       },
       "CCloudConfig" : {
         "description" : "Configuration for Confluent Cloud connections",
-        "required" : [ "organization_id" ],
         "type" : "object",
         "properties" : {
           "organization_id" : {
             "description" : "The identifier of the CCloud organization to use. The user's default organization is used when absent.",
             "maxLength" : 36,
             "minLength" : 36,
             "type" : "string"
+          },
+          "ide_auth_callback_uri" : {
+            "description" : "The URI that users will be redirected to after successfully completing the authentication flow with Confluent Cloud.",
+            "maxLength" : 200,
```

Comment:
curious about why we chose 200? Thinking about 3.2.1 General Syntax on [this page ](http://www.faqs.org/rfcs/rfc2616.html)

Reply:
I chose it more or less randomly. All redirect URIs that I can think of have less than 200 characters. At the same time, this validation prevents passing very long (invalid?) strings via the `ide_auth_callback_uri`.

We can (slightly) decrease the `maxLength` constraint to , for instance, 100 or 150 characters; LMK if you want me to do so.

Reply:
I think 200 is fine